fastlane/Fastfile

```diff
@@ -677,6 +686,41 @@ platform :ios do
     )
   end
 
+  #####################################################################################
+  # upload_gutenberg_sourcemaps
+  # -----------------------------------------------------------------------------------
+  # This lane uploads the Gutenberg source maps
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane upload_gutenberg_sourcemaps [internal:<internal>]
+  #
+  # Example:
+  # bundle exec fastlane upload_gutenberg_sourcemaps internal:true
+  #####################################################################################
+  lane :upload_gutenberg_sourcemaps do | options |
+    build_version = ios_get_build_version(internal: options[:internal])
+    
+    gutenberg_bundle = File.join(PROJECT_ROOT_FOLDER, "Pods/Gutenberg/bundle/ios")
+    sourcemaps_folder = File.join(gutenberg_bundle, "sourcemaps")
+    FileUtils.mkdir_p(sourcemaps_folder)
+    # Name of bundle file must be set to main.jsbundle
+    FileUtils.cp(File.join(gutenberg_bundle, 'App.js'), File.join(sourcemaps_folder, 'main.jsbundle'))
+    FileUtils.cp(File.join(gutenberg_bundle, 'App.js.map'), File.join(sourcemaps_folder, 'main.jsbundle.map'))
+
+    sentry_upload_sourcemap(
+      auth_token: get_required_env("SENTRY_AUTH_TOKEN"),
+      org_slug: 'a8c',
+      project_slug: 'wordpress-ios',
+      version: build_version,
+      dist: build_version,
+      rewrite: true,
+      strip_common_prefix: true,
+      sourcemap: sourcemaps_folder
+    )
+
+    FileUtils.rm_rf(sourcemaps_folder)
```

Comment:
Ruby suggestion: there's a way to run this code that depends on a temporary directory in a block, so that even if something fails, the runtime will ensure the directory is removed.

It should be something like:

```ruby
require 'tmpdir'

Dir.mktmpdir do |sourcemaps_folder|
  # all the code should "just work" the way it's written right now
end
```

Reply:
Thanks for the suggestion 🙇 ! I'll update the code to use the temporary directory block.

Reply:
I applied your suggestion in [this commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/16700/commits/20f1892b50477fb1a415e0991e7135c5e99a1aaf) and run some tests, it worked like charm 🎊 .